src/js/components/VoteTable/VoteCalendarLink.vue

```diff
@@ -0,0 +1,31 @@
+<script>
+
+export default {
+	name: 'VoteCalendarLink',
+
+	methods: {
+		newEvent(option) {
+			const dateTimeStart = option.timestamp
+			const dateTimeEnd = moment.unix(dateTimeStart).add(1, 'hour').unix()
+
+			const protocol = window.location.protocol + '//'
+			const host = window.location.host
+			// Useful if the current pathname is '/index.php/apps/...'
+			const pathname = window.location.pathname.match('^(.*?/apps)/.*$')
+			const appsRoot = pathname ? pathname[1] : '/apps'
+
+			const calendarRoot = appsRoot + '/calendar'
```

Comment:
NC already has a function to get the URL of an app, so you don't have to fiddle with `window.location`. It's simply `OC.generateUrl('calendar');` :)